### ISSUE TITLE
Meteor Framework support

### ISSUE THREAD

**Issue by chicagogrooves** [tastejs/todomvc#1124]
The Support for the [Meteor](www.meteor.com) framework is in need of updating in two ways

* The existing demo of full server-hosted Meteor is outdated
* I've extracted the client-side parts of Meteor to their own repo (https://github.com/chicagogrooves/mini-meteor, on bower and NPM) which could just do the client side templating..

I'm willing to help on one or more of these fronts, let's stay updated..

**Comment by sindresorhus** [comment]
> The existing demo of full server-hosted Meteor is outdated

I'm trying to figure out who has access to it.

> I've extracted the client-side parts of Meteor to their own repo (https://github.com/chicagogrooves/mini-meteor, on bower and NPM) which could just do the client side templating..

I don't think we want to diverge too much from the original. Happy to link to an example using it from the [wiki](https://github.com/tastejs/todomvc/wiki/Other-implementations) though.

> I'm willing to help on one or more of these fronts, let's stay updated..

We could use some help updating the Meteor app to our new UI if you're interested. https://github.com/tastejs/todomvc/issues/1110

**Comment by sindresorhus** [comment]
@passy Did you get Meteor installed and checked if you could `claim` it?

**Comment by chicagogrooves** [comment]
I tried, but could not `claim` it with the basic command. And `meteor authorized` wouldn't tell me who owned it. I'll check out #1110 and see if I can put up a parallel domain, would that help ?

**Comment by sindresorhus** [comment]
> I'll check out #1110 and see if I can put up a parallel domain, would that help ?

Yes, but I would prefer for us to have todomvc.meteor.com if possible.

**Comment by arthurvr** [comment]
Also related: https://github.com/tastejs/todomvc/pull/1316

**Comment by addyosmani** [comment]
Thanks so much for this contribution. I'm going through old issues and closing them out as we last updated the project in a major way in 2024 and want to ensure we're focused on only those issues still relevant. Thanks once again, feel free to reopen if still relevant
